### ISSUE TITLE
tips to improve flow training

### ISSUE THREAD

**Issue by taalua** [FunAudioLLM/CosyVoice#252]
Hi, 

I am training flow from the scratch using higher number of mel bins, 
it works, but the sound quality is not as good as the pretrained model, any tips to train flow model for improved quality of sound?
Also, I found out that after a certain number of epochs, the loss explodes.
Thanks.

**Comment by aluminumbox** [comment]
use more data, reduce lr if the loss explodes
